src/Components/drills/FrisbeeDrillIllustration.js

```diff
@@ -94,8 +94,8 @@ const FrisbeeDrillIllustration = (props) => {
     return (
       <>
         {renderTitle(item.title, index)}
-        <View style={styles.pagination}>{renderPagination()}</View>
         <ScrollView>
+          <View style={styles.pagination}>{renderPagination()}</View>
```

Comment:
Are you sure about this change?

In minimaldrillpage,, if i scroll down to read the text, I don't see the dots anymore:

![Capture du 2020-11-11 23-10-48](https://user-images.githubusercontent.com/3810795/98870206-41e07480-2473-11eb-9f40-a78da48153b0.png)

Reply:
no It was a mistake that I have to change. thks Martin